bite-size-notes/week-8-a-Probability Review-Zhijie Tan

```diff
@@ -0,0 +1,204 @@
+## Probability Review
+## Scope
+Probability is a branch of mathematics that deals with the occurrence of random events. It has been introduced in math to predict how likely events are to happen. The meaning of probability is essentially the extent to which something is likely to happen.[1] In probability theory, it is also used in probability distributions, which show the possibility of random cases. Probability theory is a fundamental aspect of statistical analysis and is used in various fields, including science, engineering, economics, and social sciences.
+
+This section covers the introduction of symbols related to Probability and discusses Random variables in different formulas. 
+
+##
+The following notes will explain the meanings of symbols and demonstrate different formulas related to random variables.
+
+- Probability Distribution: An introduction to probability distributions, which describe how probabilities are distributed over the values of a random variable.
+
+- Example of Probability Mass Function (PMF): A discussion and example of PMFs, which apply to discrete random variables and give the probability that a discrete random variable is exactly equal to some value.
+
+- Example of Probability Density Function (PDF): A discussion and example of PDFs, which apply to continuous random variables and describe the likelihood of a random variable to take on a particular value.
+
+- Rules Corresponding to Random Variables: An overview of some fundamental rules and properties related to random variables.
+
+### Random Variable
+
+### Symbol
+- X:random varible(RV). a quantity whose value is unknown/uncertain 
+- x: possible value of X or sample of X or realization of X 
+
+
+### Probability Space ($\Omega$,$F$,$\mathcal{P}$)
+- $\Omega$: sample space (all possible outcomes of RV)
+- $\mathcal{F}$: Event space (all possible sets of outcomes in $\Omega$)
+- $\mathcal{P}$: Probability measure (assigns a probability to each other)
+
+## Moving forward consider X v P(X) where P(x) is a probabilty distribution
+- Expectation $\mathbb{E}$
+  
+  ${\mathbb{E} \text{xp} [g(x)]} = \int_x g(x) P(x) \, dx \text{ or } \sum_x g(x) P(x)$
+  - mean $\mathcal{u_x}$ = ${\mathbb{E} \text{xp}(x) [g(x)]}$
+
+
+
+### Discrete Random Variable Plot
+
+The following plot represents the probability mass function (PMF) of a discrete random variable: 
+![Discrete Random Variable Plot](PMF.png)
+- $S=[1.2.3]$
```

Comment:
’$S=[1.2.3]$‘ should be ’$S = {1, 2, 3}$‘

Reply:
Fixed. Thank you!